index.bs

```diff
@@ -1218,7 +1218,7 @@ following Web IDL.
 </pre>
 
 <div dfn-type="enum-value" dfn-for="PublicKeyCredentialType">
-    This enumeration defines the valid credential types. It is an extension point; values may be added to it in the future, as
+    This enumeration defines the valid credential types. It is an extension point; values MAY be added to it in the future, as
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
changed to can.